src/utils.ts

```diff
@@ -1,30 +1,25 @@
+import { PullRequest } from "./github";
+
 /**
- * @param template The backport description template
+ * @param template The template
  * @param main The main pull request that is backported
  * @param target The target branchname
- * @returns Description that can be used as the backport pull request body
+ * @returns Description that can be used in the backport pull request
  */
-export function composeBody(
+export function composeMessage(
```

Comment:
I have renamed this function to have a more generic name (used as title or description) but `message` could be also too specific, so I am open to suggestions

Reply:
fixed